tests/events/listStatusChanges.test.js

```diff
@@ -38,3 +38,38 @@ test('properties of status change', async () => {
     expect(statusChange.value.eventId).toBe(event.id);
     expect(statusChange.value.extraData).toEqual({'foo': 'bar'});
 });
+
+test('that status change contains holdToken', async () => {
+    let chartKey = testUtils.getChartKey();
+    await testUtils.createTestChart(chartKey, user.designerKey);
+    let event = await client.events.create(chartKey);
+    let holdToken = await client.holdTokens.create();
+    await client.events.hold(event.key, 'A-1', holdToken.holdToken);
+    await client.events.book(event.key, 'A-1', holdToken.holdToken);
```

Comment:
don't think you need to book in this test, since you're only asserting that the first status change has a hold token

Reply:
Very true